c-deps/libroach/db.cc

```diff
@@ -43,6 +43,9 @@ static void __attribute__((noreturn)) die_missing_symbol(const char* name) {
   abort();
 }
 
+// OpenHook does nothing in OSS mode.
```

Comment:
This is much better than my previous method (see first commit). Thanks @benesch!